README.md

```diff
@@ -46,6 +46,22 @@ Starkup is created by the same team at [Software Mansion] that is behind [Scarb]
 
 Feel free to chat with us on our channel on [Telegram] or [Starknet's Discord]!
 
+## Troubleshooting
+
+If you have `curl` installed through `snap` store, Starkup may fail with following error:
```

Comment:
nit: we could also hint users on how to do this?

```
$ sudo snap list | grep curl
```

should do the trick

Reply:
good point! I'll add it